scylla/src/transport/connection.rs

```diff
@@ -2479,6 +2478,8 @@ mod tests {
             .query_iter(select_query.clone())
             .await
             .unwrap()
+            .rows_stream::<Row>()
+            .unwrap()
             .try_collect::<Vec<_>>()
             .await
             .unwrap();
```

Comment:
Why do you use `Row`? It looks like the types are known here.

Reply:
The old code used untyped API, that's why I resorted to `Row`. I didn't want to invest my time in checking what's the expected type of this row.